app/Providers/RouteServiceProvider.php

```diff
@@ -24,7 +24,7 @@ public function boot()
 
             $this->mapWebRoutes();
 
-            //
+            require base_path('routes/scim.php'); //This 'require' thing feels janky, but, well, whatever. I dunno
```

Comment:
That’s how Laravel wants it, so janky or not, I think this is correct for the purpose

Reply:
Ok, will pull or adjust the comment.